project/ApiDefinitionsGenerator.scala

```diff
@@ -0,0 +1,164 @@
+
+import scala.collection.mutable.ArrayBuffer
+import scala.util.parsing.json.JSON
+import scala.io.Source
+object ApiDefinitionsGenerator {
+
+  def modules(): Seq[module] = {
+    //todo: read term_info.json from java driver
+    val jsonFilename = "term_info.json"
+    var modules = ArrayBuffer[module]()
```

Comment:
1. Don't use `var` for mutable collection definition.
2. Don't use mutable collections for parsing cases :)